plugins/outputs/sematext/processors/processor.go

```diff
@@ -0,0 +1,7 @@
+package processors
+
+import "github.com/influxdata/telegraf"
+
+type Processor interface {
```

Comment:
no comments on exported types? Maybe Telegraf doesn't impose lint rules for this?

Reply:
Will add where missing.

Reply:
Should we adhere to the https://github.com/sematext/telegraf/blob/6b8d3601fe42413ed934c64a4ca69930a86b7e83/processor.go#L6 interface instead of rolling our own?

Reply:
It is an intriguing thought, but I'd prefer to keep our own interface:
- it would give us more flexibility because we'd be able to change input/output
- I wouldn't want anyone else to confuse our processor implementations (since they would implement the interface) with processors that could be used in Telegraf